backend/src/main/java/me/performancereservation/api/RefundController.java

```diff
@@ -0,0 +1,72 @@
+package me.performancereservation.api;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import me.performancereservation.domain.refund.RefundService;
+import me.performancereservation.domain.refund.dto.RefundDetailResponse;
+import me.performancereservation.domain.refund.dto.RefundRequest;
+import me.performancereservation.domain.refund.enums.RefundStatus;
+import me.performancereservation.global.exception.AppException;
+import me.performancereservation.global.exception.ErrorCode;
+import me.performancereservation.global.exception.ErrorType;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@Slf4j
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/v1/refunds")
+public class RefundController {
+
+    private final RefundService refundService;
+
+    /*--- USER 요청에 대응 ---*/
+    // 본인 id와 일치하는 모든 환불내역 리스트 반환
+    @GetMapping("/user/{userId}")
```

Comment:
```suggestion
    @GetMapping("/me")
```
본인 거를 조회하거나 다루는 api는 RESTful 관점에서 me를 관례적으로 사용하는 것으로 알고있습니다

Reply:
오호 처음 알게 된 사실!

Reply:
url에 사용자아이디를 노출시키지 않기 위해 아래의 코드로 user id를 불러오려고 하는데 아직 로그인이 구현 전이라.. 임시로 /uesr/{userId}로 정보를 받게 써두고 나중에 수정해도 될까요?
```java
Long userId = ((CustomUserDetails) SecurityContextHolder.getContext().getAuthentication().getPrincipal()).getId();
```

Reply:
넵 좋습니다!
저렇게 수동으로 추출도 가능한데 
@AuthenticationPrincipal 파라미터로 authentication.userId()같은걸로 뽑을 수 있어요

Reply:
오 좋아보입니다 나중에 수정할 때 참고하겠습니다!